React/Views/RCTSlider.m

```diff
@@ -32,4 +32,9 @@ - (void)setMaximumValue:(float)maximumValue
   super.value = _unclippedValue;
 }
 
+- (void)setThumbImage:(UIImage *) thumbImage
```

Comment:
You'll need to add a getter for this as well to prevent crashes if you pass null for the image:

    - (UIImage *)thumbImage
    {
      return [self thumbImageForState:UIControlStateNormal];
    }